src/sentry/utils/snuba.py

```diff
@@ -44,6 +44,7 @@
     # general
     'id': 'event_id',
     'project.id': 'project_id',
+    'type': 'type',
```

Comment:
We already have this as `event.type`. Looking back at the history of this, we did it because existing users are already using `type` as a tag, and this would override that. Better to just use `event.type` in discover, since it already works in event/issue search.

Reply:
@saragilford Any thoughts on naming here?

Reply:
I think because we have so many projects using `type`, I'm hesitant to ask people to change their tag usage (anything that requires our users to make code changes is a hard sell). While `event.type` is not the most elegant, it's the right choice for us to commit to.